src/index.js

```diff
@@ -185,65 +186,81 @@ function toProto(doc) {
   // params
   coll = doc.alps.descriptor.filter(semantic);
   coll.forEach(function(msg) {
-    rtn += `message ${msg.id}Params {\n`;
+    val = makeSnakeCase(msg.id);
```

Comment:
`makePascalCase()` should be applied here

Reply:
OK. will make this change.-- #thanks

Reply:
@corntoole : 

pretty sure these changes resullt in an invalid proto file:

```
syntax = "proto3";
package ToDo_API;

// *******************************************************************
// generated by "unified" from todo-alps.yaml
// date: Thu Dec 31 2020 19:28:00 GMT-0500 (Eastern Standard Time)
// http://github.com/mamund/2020-11-unified
// *******************************************************************

message Id_params {
  string Id = 1;
}
message Body_params {
  string Body = 1;
}

message TodoItem {
  string id = 1;
  string body = 2;
}
message TodoItem_response {
  repeated TodoItem TodoItem_collection = 1;
}
message TodoItem_empty {}

service TodoApiService {
  rpc TodoList(todo_item_empty) returns (todo_item_response) {};
  rpc TodoAdd(todo_item) returns (todo_item_response) {};
  rpc TodoRemove(id_params) returns (todo_item_response) {};
}
```

wdyt?